controllers/flinkcluster_converter.go

```diff
@@ -330,43 +312,19 @@ func getDesiredTaskManagerDeployment(
 	var rpcPort = corev1.ContainerPort{Name: "rpc", ContainerPort: *taskManagerSpec.Ports.RPC}
 	var queryPort = corev1.ContainerPort{Name: "query", ContainerPort: *taskManagerSpec.Ports.Query}
 	var taskManagerDeploymentName = getTaskManagerDeploymentName(clusterName)
-	var jobManagerDeploymentName = getJobManagerDeploymentName(clusterName)
 	var labels = map[string]string{
 		"cluster":   clusterName,
 		"app":       "flink",
 		"component": "taskmanager",
 	}
-	var envVars = []corev1.EnvVar{
-		{
-			Name:  "JOB_MANAGER_RPC_ADDRESS",
-			Value: jobManagerDeploymentName,
-		},
-		{
-			Name: "TASK_MANAGER_CPU_LIMIT",
-			ValueFrom: &corev1.EnvVarSource{
-				ResourceFieldRef: &corev1.ResourceFieldSelector{
-					ContainerName: "taskmanager",
-					Resource:      "limits.cpu",
-					Divisor:       resource.MustParse("1m"),
-				},
-			},
-		},
-		{
-			Name: "TASK_MANAGER_MEMORY_LIMIT",
-			ValueFrom: &corev1.EnvVarSource{
-				ResourceFieldRef: &corev1.ResourceFieldSelector{
-					ContainerName: "taskmanager",
-					Resource:      "limits.memory",
-					Divisor:       resource.MustParse("1Mi"),
-				},
-			},
-		},
-		{
-			Name:  "FLINK_PROPERTIES",
-			Value: getFlinkProperties(flinkCluster.Spec.FlinkProperties),
-		},
-	}
-	envVars = append(envVars, flinkCluster.Spec.EnvVars...)
+	// Make Volume, VolumeMount to use configMap data for flink-conf.yaml, if flinkProperties is provided.
```

Comment:
Here it says "if flinkProperties is provided", but it seems not true, configMap is always created, is my understanding correct?

Reply:
The configmap is always created. I'll edit the comment.